jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/api/JibContainerBuilderTestHelper.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Should probably move this to core or plugins-common, I think we'll need it for Maven as well.

Reply:
Or possibly use an alternative design that doesn't involve wrapping a single method, but I'm not sure the best way of doing that.

Reply:
Will move to `plugins-common`. This was mostly necessary because didn't want to put an `@VisibleForTesting` method in the user-exposed `JibContainerBuilder` class.